### ISSUE TITLE
[CAIA A11y Support]: Benefits team, Responsive tables on MacOS VoiceOver bug

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#58146]
### What do you need support with?

Need help auditing the responsive table web component on MacOS using VoiceOver on Safari. [Specifically, we need an accessibility specialist to validate the following issue.](https://cssence.com/2020/voiceover-table-issue/)

If validated, please

1. Submit a bug ticket with Apple
2. Alert the design system team of the issue

### When do you need this support by?

May 30, 2023

### How urgent is this request?

A little urgent, we need this support before the date we provided above

### Supporting artifacts (if any)

[Table web component](https://design.va.gov/storybook/?path=/story/components-va-table--default)

### Who's on your team?

Team name: Benefits team
OCTO-DE product owner: Matt Self
Product manager: Jacob Worrell
Designer: ??
FE engineer: ??
Product/team Slack channel: ??

### Do you have an accessibility champion on your team?

Yes, we have a champion

### If you have an accessibility champion, what's their name?

Robin Garrison

### Is anyone on your team interested in becoming an accessibility champion?

Several!

**Comment by joshkimux** [comment]
[Closing, as an issue has been submitted to Apple](https://bugs.webkit.org/show_bug.cgi?id=257458)
